rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -314,7 +314,9 @@ rmw_create_client(
     reader_qos.data_sharing().off();
   }
 
-  if (!get_datareader_qos(adapted_qos_policies, reader_qos)) {
+  if (!get_datareader_qos(
+      adapted_qos_policies, *type_supports->response_typesupport->type_hash, reader_qos))
```

Comment:
nitpick: I don't know if this package has our linters run over it, but I would have expected this to use two spaces of indent, not four

Reply:
linters did run - your intuition is my intuition, but `uncrustify` wants it this way. I believe the logic is "2 times the  number of opening braces" - in this case there's `if ((` so it expects 4 spaces